docs/groundstate.rst

```diff
@@ -295,7 +295,31 @@ find the energy.
 
 Go to :ref:`top <groundstate>`.
 
-.. _ gs_scf_adv:
+.. _gs_scf_restart:
+
+Restarting SCF
+~~~~~~~~~~~~~~
+
+The SCF cycle can be restarted from a previous density matrix or
+charge density, which may significantly speed up convergence.
+The density matrix is automatically written out in the files ``Kmatrix2.*`` or
+``Lmatrix2.*`` (depending on whether diagonalisation or linear scaling
+is being used).  These files are read in, and the initial
+charge density made from them by setting the flags:
+
```

Comment:
Thank you very much for adding this comment. 
Users can also write out  the matrix files like``Kmatrix2.iXX.p****`` and ``InfoGlobal.iXX``during the DMM, SCF or opt-Support steps by setting ``IO.DumpFreq.DMM``,``IO.DumpFreq.SCF``, ``IO.DumpFreq.SFcoeff``.  These files can be read by setting ``IO.MatrixFile.FileIndex XX``.  
But, I think it is probably better not to explain these input parameters because it is tricky..